raco/algebra.py

```diff
@@ -1013,6 +1013,13 @@ def scheme(self):
             schema.addAttribute(name, _type)
         return schema
 
+    def __eq__(self, other):
```

Comment:
I guess we have to implement `__ne__` as well, right? In the immortal words of the [documentation](https://docs.python.org/2/reference/datamodel.html#object.__ne__): "There are no implied relationships among the comparison operators. The truth of `x==y` does not imply that `x!=y` is false."

Sometimes I think the creator of PHP had more design sense than Guido...